crates/blockchain/error.rs

```diff
@@ -32,6 +32,10 @@ pub enum InvalidBlockError {
     GasUsedMismatch,
     #[error("Blob gas used doesn't match value in header")]
     BlobGasUsedMismatch,
+    #[error(
+        "Attempted to add an invalid transaction to the block. The transaction filter must have failed."
+    )]
+    AttemptedToAddInvalidTransaction,
```

Comment:
the error is a bit long. Why not just `InvalidTransaction`?

Reply:
Not a problem, renaming it asap.

Reply:
Done [6f3a2fb](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1018/commits/6f3a2fbdf00a4ad6a87cfac57271f09a27d7d2ef)